src/js/components/Main/Home/Home.container.jsx

```diff
@@ -0,0 +1,38 @@
+import React, {useState, useEffect} from 'react';
+import Home from './Home.jsx'
+
+
+function HomeContainer(props) {
+
+  const [inputText, setInputText] = useState("");
+  const [concerts, setConcerts] = useState([]);
+  const [helpList, setHelpList] = useState([]);
+
+  useEffect(() => {
+    props.getConcerts(setConcerts)
+  })
```

Comment:
can you please install `https://www.npmjs.com/package/eslint-plugin-react-hooks`
since we want to use hooks properly